security/appsec/intro/what-else-do-you-need-to-know.md

```diff
@@ -68,3 +62,11 @@ How memory in a computer works, and where data is located at any given time
 * Binary
 * Hexadecimal
 * Cryptography
+
+---
+## Footnotes
+
+[1:Levels of AppSec]
+It's something everyone needs to know at some level, so that when major outbreaks occur they can take steps to protect themselves and the applications they develop
```

Comment:
It also means knowing how to build a secure application from the start, not just when major outbreaks occur